### PR TITLE
docs(component): Update to IDF 5.1 and add component manager instructions

### PR DESCRIPTION
**Pull request by P-R-O-C-H-Y**
## Description of Change
This PR updates the documentation page of using Arduino framework as ESP-IDF component.

### CODE REVIEW

**Review by me-no-dev — APPROVED** [review]

File: docs/en/esp-idf_component.rst [hunk 1]
```diff
@@ -14,14 +14,36 @@ For a simplified method, see `Installing using Boards Manager <https://docs.espr
...
 Installation
 ------------
 
-.. note:: Latest Arduino Core ESP32 version is now compatible with `ESP-IDF v4.4 <https://github.com/espressif/esp-idf/tree/release/v4.4>`_. Please consider this compatibility when using Arduino as a component in ESP-IDF.
-
 #. Download and install `ESP-IDF <https://github.com/espressif/esp-idf>`_.
 
    * For more information see `Get Started <https://docs.espressif.com/projects/esp-idf/en/latest/esp32/get-started/index.html#installation-step-by-step>`_.
+
+Install using IDF Component Manager
```
Comment (VojtechBartoska) [inline docs/en/esp-idf_component.rst:29]: Installing

File: docs/en/esp-idf_component.rst [hunk 2]
```diff
@@ -14,14 +14,36 @@ For a simplified method, see `Installing using Boards Manager <https://docs.espr
...
+***********************************
+
+To add the Arduino component to your project using the IDF Component Manager, run the following command in your project directory:
+
+.. code-block:: bash
+
+    idf.py add-dependency "espressif/arduino-esp32^3.0.2"
+
+Or you can start a new projetc from a template with the Arduino component:
```
Comment (VojtechBartoska) [inline docs/en/esp-idf_component.rst:38]: typo: project

**Review by VojtechBartoska — APPROVED** [review]
typo fix

**Comment by P-R-O-C-H-Y** [comment]
@VojtechBartoska Typos fixed. Marking as pending merge.

**Pull request merged**